File: src/main/java/um/haberes/report/model/haberes/core/LegajoBancoDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LegajoBancoDto {

    private Long legajoBancoId;

    private Long legajoId;

    private int anho = 0;

    private int mes = 0;

    private String cbu = "";

    private BigDecimal fijo = BigDecimal.ZERO;

    private BigDecimal porcentaje = BigDecimal.ZERO;

    private byte resto = 0;

    private BigDecimal acreditado = BigDecimal.ZERO;

    private PersonaDto persona;

    private LiquidacionDto liquidacion;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/LegajoBancoDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.math.BigDecimal

data class LegajoBancoDto(

    var legajoBancoId: Long? = null,
    var legajoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var cbu: String = "",
    var fijo: BigDecimal = BigDecimal.ZERO,
    var porcentaje: BigDecimal = BigDecimal.ZERO,
    var resto: Byte = 0,
    var acreditado: BigDecimal = BigDecimal.ZERO,
    var persona: PersonaDto? = null,
    var liquidacion: LiquidacionDto? = null

)
